pkg/controller/volume/pvcprotection/pvc_protection_controller_test.go

```diff
@@ -388,7 +500,7 @@ func TestPVCProtectionController(t *testing.T) {
 		if !test.informersAreLate {
 			informersObjs = append(informersObjs, test.initialObjects...)
 		}
-
+		fmt.Printf("Test name: %v\n", test.name)
```

Comment:
I'd move this printf up just under the for loop definition. Also, if appropriate use the klog library, rather than fmt

Reply:
I was only adding this during debugging probably, so we can just delete it now.